### DISCUSSION TITLE
Provision of Pretix supported MTN MOMO and AIRTEL MONEY payment providers

### DISCUSSION THREAD

**Discussion by akolaarthurali** [pretix/pretix#4030] [General]
It is very understandable an E-commerce platform just like pretix is cannot better serve it mission without a payment system **_that is popular amongst its audience._** 

I think pretix has served its European audience better with provision of stripe and PayPal as the default payment systems in pretix.  

However it is quite unfortunate that such payments providers aren't popular among many African countries who conveniently prefer **MOBILE  MONEY  solutions (MTN MOMO, AIRTEL Mobile )** to  transfer  money and make payment of goods and services via a USSD Code).

It is from that need I **humbly request fellow  developers here dillgently contribute towards to the design and development  of such payment plugins** to serve our African audiences so they too can benefit from massive potential of pretix. 

I duly understand the pretix_MTN_MOMO exists on our pretix marketplace but i, personally haven't had access to it.
 
**My role and responsibility to this initiative** 
I am more than willing  to offer an extensive knowledge base on how to ACCESS THE MTN and Airtel Mobile Money Global APIs

**Comment by raphaelm** [comment]
We have decided to release our MTN Mobile Money implementation as open source: https://github.com/pretix/pretix-mtn-momo

**Reply by akolaarthurali** [reply]
Thank you so much @raphaelm. 😀 thanks for being so generous. I will definitely go ahead use it.
